docs/guides/setup/configuring-metadata-stores/how-to-configure-a-validation-result-store-in-gcs.md

```diff
@@ -1,5 +1,108 @@
+                project: '<your_GCP_project_name>'
+                bucket: '<your_GCS_bucket_name>'
+                prefix: '<your_GCS_folder_name>'
+    ```
+
+:::warning
+If you are also storing [Expectations in GCS](../configuring-metadata-stores/how-to-configure-an-expectation-store-in-gcs.md) or [DataDocs in GCS](../../../configuring-data-docs/how-to-host-and-share-data-docs-on-gcs.md), please ensure that the ``prefix`` values are disjoint and one is not a substring of the other.
+:::
+
+
+4. **Copy existing Validation results to the GCS bucket**. (This step is optional).
+
+    One way to copy Validations into GCS is by using the ``gsutil cp`` command, which is part of the Google Cloud SDK. In the example below, two Validation results, ``Validation1`` and ``Validation2`` are copied to the GCS bucket.   Information on other ways to copy Validation results, like the Cloud Storage browser in the Google Cloud Console, can be found in the [Documentation for Google Cloud](https://cloud.google.com/storage/docs/uploading-objects).
+
+    ```bash
+    gsutil cp uncommitted/validations/Validation1.json gs://'<your_GCS_bucket_name>'/'<your_GCS_folder_name>'
+    gsutil cp uncommitted/validations/Validation2.json gs://'<your_GCS_bucket_name>'/'<your_GCS_folder_name>'
+
+    Operation completed over 2 objects/58.8 KiB.
+    ```
+
+
+
+5. **Confirm that the new Validations store has been added by running** ``great_expectations --v3-api store list``.
+
+    Notice the output contains two Validation stores: the original ``validations_store`` on the local filesystem and the ``validations_GCS_store`` we just configured.  This is ok, since Great Expectations will look for Validations in GCS as long as we set the ``validations_store_name`` variable to ``validations_GCS_store``, and the config for ``validations_store`` can be removed if you would like.
+
+    ```bash
+    great_expectations --v3-api store list
+
+    - name: validations_store
+    class_name: ValidationsStore
+    store_backend:
+        class_name: TupleFilesystemStoreBackend
+        base_directory: uncommitted/validations/
+
+    - name: validations_GCS_store
+    class_name: ValidationsStore
+    store_backend:
+        class_name: TupleGCSStoreBackend
+        project: '<your_GCP_project_name>'
+        bucket: '<your_GCS_bucket_name>'
+        prefix: '<your_GCS_folder_name>'
+    ```
+
+6. **Confirm that the Validations store has been correctly configured.**
+
+    Run a [Checkpoint](../../../tutorials/getting-started/validate-your-data.md) to store results in the new Validations store on GCS then visualize the results by re-building [Data Docs](../../../tutorials/getting-started/validate-your-data.md).
+
+
+If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.  Also, please reach out to us on [Slack](https://greatexpectations.io/slack) if you would like to learn more, or have any questions.
```

Comment:
Do we have a comments section in our new docs? I think the link to Slack is relevant but the `+1` and suggestions/questions below part should be omitted right?

Reply:
I think its ok since that is where Kyle looks